src/patternfly/components/CalendarMonth/calendar-month.scss

```diff
@@ -0,0 +1,226 @@
+  --pf-c-calendar-month__grid-cell--before--Bottom: var(--pf-c-calendar-month__grid-body__grid-cell--PaddingBottom);
+  --pf-c-calendar-month__grid-cell--before--Left: 0;
+  --pf-c-calendar-month__grid-cell--m-in-range--before--BackgroundColor: var(--pf-global--palette--blue-50);
+  --pf-c-calendar-month__grid-cell--m-in-range--m-start-range--before--Left: 50%;
+  --pf-c-calendar-month__grid-cell--m-in-range--m-end-range--before--Right: 50%;
+  --pf-c-calendar-month__grid-cell--m-adjacent-month__date--Color: var(--pf-global--disabled-color--100);
+
+  // date
+  --pf-c-calendar-month__date--Width: 4ch;
+  --pf-c-calendar-month__date--Height: 4ch;
+  --pf-c-calendar-month__date--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-calendar-month__date--Color: var(--pf-global--Color--100);
+  --pf-c-calendar-month__date--BackgroundColor: transparent;
+  --pf-c-calendar-month__date--disabled--Color: var(--pf-global--disabled-color--200);
+  --pf-c-calendar-month__date--after--BorderWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-calendar-month__date--after--BorderColor: transparent;
+  --pf-c-calendar-month__date--hover--after--BorderColor: var(--pf-global--active-color--100);
+  --pf-c-calendar-month__date--focus--after--BorderColor: var(--pf-global--active-color--100);
+  --pf-c-calendar-month__grid-cell--m-start-end-range__date--hover--BackgroundColor: var(--pf-c-calendar-month--BackgroundColor);
+  --pf-c-calendar-month__grid-cell--m-start-end-range__date--focus--BackgroundColor: var(--pf-c-calendar-month--BackgroundColor);
+
+  @include pf-t-light;
+
+  display: inline-flex;
+  flex-direction: column;
+  padding: var(--pf-c-calendar-month--PaddingTop) var(--pf-c-calendar-month--PaddingRight) var(--pf-c-calendar-month--PaddingBottom) var(--pf-c-calendar-month--PaddingLeft);
+  font-size: var(--pf-c-calendar-month--FontSize);
+  background-color: var(--pf-c-calendar-month--BackgroundColor);
+}
+
+.pf-c-calendar-month__header {
+  display: flex;
+  margin-bottom: var(--pf-c-calendar-month__header--MarginBottom);
+}
+
+.pf-c-calendar-month__header-nav {
+  margin-right: var(--pf-c-calendar-month__header-nav--MarginRight);
+  margin-left: var(--pf-c-calendar-month__header-nav--MarginLeft);
+
+  &.pf-m-prev-month {
+    --pf-c-calendar-month__header-nav--MarginRight: var(--pf-c-calendar-month__header-nav--m-prev-month--MarginRight);
+    --pf-c-calendar-month__header-nav--MarginLeft: var(--pf-c-calendar-month__header-nav--m-prev-month--MarginLeft);
+  }
+
+  &.pf-m-next-month {
+    --pf-c-calendar-month__header-nav--MarginRight: var(--pf-c-calendar-month__header-nav--m-next-month--MarginRight);
+    --pf-c-calendar-month__header-nav--MarginLeft: var(--pf-c-calendar-month__header-nav--m-next-month--MarginLeft);
+  }
+}
+
+.pf-c-calendar-month__header-month {
+  flex-grow: 1;
+}
+
+.pf-c-calendar-month__header-year {
+  width: var(--pf-c-calendar-month__header-year--Width);
+}
+
+.pf-c-calendar-month__body {
+  display: flex;
```

Comment:
are these needed ?

Reply:
It will keep the calendar part centered if for some reason the container is wider than the calendar, and will keep react styles happy by giving this element styles, but you're right - it's probably better to do it another way. Aligning everything in `pf-c-calendar-month` works better.